src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -1589,12 +1640,13 @@ void validateLifetimeOfReferences(BasicBlock block)
 
                 if (referencedLocalsAndMethods.Count != 0)
                 {
-                    Assert.True(false, $"Local/method without owning region {referencedLocalsAndMethods.First().ToTestDisplayString()} in [{getBlockId(block)}]");
+                    ISymbol symbol = referencedLocalsAndMethods.First();
+                    Assert.True(false, $"{(symbol.Kind == SymbolKind.Local ? "Symbol" : "Method")} without owning region {symbol.ToTestDisplayString()} in [{getBlockId(block)}]\n{finalGraph}");
                 }
 
                 if (referencedCaptureIds.Count != 0)
                 {
-                    Assert.True(false, $"Capture [{referencedCaptureIds.First().Value}] without owning region in [{getBlockId(block)}]");
+                    Assert.True(false, $"Capture [{referencedCaptureIds.First().Value}] without owning region in [{getBlockId(block)}]\n{finalGraph}");
```

Comment:
>\n{finalGraph} [](start = 134, length = 14)

We are always building a string that cold be quite long. Is there a version of Assert that can do this lazily?